io.mpv.Mpv.yml

```diff
@@ -649,6 +658,14 @@ modules:
           version-pattern: bc-(\d\.\d+\.?\d*).tar.gz
           url-template: https://ftp.gnu.org/gnu/bc/bc-$version.tar.gz
 
+  - name: xdg-user-dirs
```

Comment:
This is available in runtime and shouldn't be needed

Reply:
When I built it without this, I couldn't find it in the `buildir/files/bin`. But I now see that this is not the whole environment. First time using `flatpak-builder`, thanks.